src/effects/RealtimeEffectList.h

```diff
@@ -54,12 +74,25 @@ class RealtimeEffectList final
    //! Apply the function to all states sequentially.
    void Visit(StateVisitor func);
 
-   //! Returns null if the id is nonempty but no such effect was found
+   //! Returns null if no such effect was found.
+   //! Sends message Insert message on success.
    RealtimeEffectState *AddState(const PluginID &id);
+   //! On success sends Remove message.
    void RemoveState(RealtimeEffectState &state);
-   void Swap(size_t index1, size_t index2);
 
-   using States = std::vector<std::unique_ptr<RealtimeEffectState>>;
+   //! Returns total amount of effects in this list
```

Comment:
"number" would be a better word than "amount" (for a count-noun, not a mass-noun)